### PR TITLE
Add ELB V3 Loadbalancer support

### PR DESCRIPTION
**Pull request by niuzhenguo**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: openstack/elb/v3/loadbalancers/requests.go [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+}
+
+// UpdateOpts is the common options struct used in this package's Update
+// operation.
+type UpdateOpts struct {
+	// Human-readable name for the Loadbalancer. Does not have to be unique.
+	Name string `json:"name,omitempty"`
+
+	// Human-readable description for the Loadbalancer.
+	Description string `json:"description,omitempty"`
```
Comment (ShiChangkuo) [inline openstack/elb/v3/loadbalancers/requests.go:163]: suggest to change the type of Description to pointer, then we can delete the older description.

**Comment by ShiChangkuo** [comment]
@niuzhenguo Is the `Get` method missing or not supported from the cloud side ?

**Comment by niuzhenguo** [comment]
> @niuzhenguo Is the `Get` method missing or not supported from the cloud side ?

it's already in L144 :D

File: openstack/elb/v3/loadbalancers/requests.go (hunk 1 above)
Comment (niuzhenguo) [reply openstack/elb/v3/loadbalancers/requests.go]: done

**Pull request merged**